javascript-if/main.js

```diff
@@ -0,0 +1,75 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function startsWithJ(string) {
+  if (string[0] === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrive(person) {
+  if (person.age < 16) {
+    return false;
+  } else {
+    return true;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else if (pH >= 0 && pH < 7) {
+    return 'acid';
+  } else if (pH <= 14 && pH >= 8) {
+    return 'base';
```

Comment:
pH levels don't need to be whole numbers. For example, human blood is slightly basic at a pH of 7.4. Your `categorizeAcidity` function should be able to categorize that properly:

---

![image](https://user-images.githubusercontent.com/16143771/169141890-65451012-c364-4aa1-937a-7858d5787671.png)